README.md

```diff
@@ -100,13 +108,13 @@ Select 1:
 What is the difference between a fork and a clone?
 
 ```text
-Your answer here.
+A fork creates a branch, a copy of code, that you can work on without fear of damaging the master. The updated code can later be merged with the master to improve the master. A clone is more general, copying information without a way/intent to push updates to the master branch later.
```

Comment:
fork makes a copy to your github account. clone makes a copy to your local machine